### PR TITLE
Dependency Updates July Week 2

### PR DESCRIPTION
**Pull request by ujahio**
### ***Publishing a new ECR image: 3.0.7. This will require manually running docker-to-ecr.sh on the court prod account***

- Dockerfile update:
  - Terraform to 1.5.3
  - AWS CLI to 2.13.1

- [Successful migration run](https://app.circleci.com/pipelines/github/flexion/ef-cms/45568/workflows/81d094fe-0064-49a6-b4b0-a185e9d607ff)
 
## Major version update(s)

 - @aws-crypto/sha256-browser: "^5.0.0"
 - eslint-plugin-prettier: "^5.0.0"
 - nodemon: "^3.0.1"
 - postcss-preset-env: "^9.0.0"
 - prettier: "^3.0.0"
 
### Security Overrides

- "stylelint": "^15.8.0"

### CODE REVIEW

File: web-client/integration-tests/judgeActivityReportJourney.test.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ describe('Judge activity report journey', () => {
     await cerebralTest.runSequence('gotoJudgeActivityReportSequence');
 
     const { isFormPristine, reportHeader } = runCompute(
-      judgeActivityReportHelper as any,
+      judgeActivityReportHelper,
```
Comment (jasonfu9966) [inline web-client/integration-tests/judgeActivityReportJourney.test.ts:33]: is this here intentionally?

File: shared/src/business/useCases/getCasesForUserInteractor.ts
Comment (jasonfu9966) [inline shared/src/business/useCases/getCasesForUserInteractor.ts:1]: is this file change included intentionally?

**Review by jasonfu9966 — APPROVED** [review]

File: web-client/integration-tests/judgeActivityReportJourney.test.ts (hunk 1 above)
Comment (ujahio) [reply web-client/integration-tests/judgeActivityReportJourney.test.ts:33]: @jasonfu9966 Yes. Change was made to remove type error. The `any` casting isn't necessary.

File: shared/src/business/useCases/getCasesForUserInteractor.ts
Comment (ujahio) [reply shared/src/business/useCases/getCasesForUserInteractor.ts:1]: @jasonfu9966 Yes. Its intentional. Attempt to resolve a TS error

File: web-client/integration-tests/judgeActivityReportJourney.test.ts (hunk 1 above)
Comment (jasonfu9966) [reply web-client/integration-tests/judgeActivityReportJourney.test.ts:33]: sounds good!

**Review by rachelschneiderman — COMMENTED** [review]
It looks like a migration was [not actually run](https://app.circleci.com/pipelines/github/flexion/ef-cms/45560/workflows/9789a8bc-602d-42d8-9996-2b5c0ad362ee/jobs/272342) in that linked deploy, unfortunately. If you open the "Setup for blue/green migration if needed" step, it printed that it was not needed and was skipping that workflow.

**Review by rachelschneiderman — APPROVED** [review]

**Review by jimlerza — APPROVED** [review]

**Pull request merged**